src/pages/entitiesPage.js

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import MainTitle from '../components/shared/mainTitle';
+import AddButton from '../components/shared/addButton';
+import { Grid } from '@mui/material';
+import Box from '@mui/material/Box';
+import * as log from 'loglevel';
+import axios from 'axios';
+import { Trans } from 'react-i18next';
+import useNotification from '../components/shared/messages/alerts';
+import useMediaQuery from '@mui/material/useMediaQuery';
+import { useTheme } from '@mui/material/styles';
+import EntitiesFilters from '../components/entities/entitiesFilter';
+import EntitiesTable from '../components/entities/entitiesTable';
+import dayjs from 'dayjs';
+
+export default function EntitiesPage({ token, graphqlErrors, env, thisTenant, tenantValues }) {
+  typeof env === 'undefined' ? log.setDefaultLevel('debug') : log.setLevel(env.LOG_LEVEL);
+
+  const [msg, sendNotification] = useNotification();
+  log.debug(msg);
+
+  const [createOpen, setCreateOpen] = React.useState(false);
+  const [entities, setEntities] = React.useState([]);
+  const mainTitle = <Trans>entities.page.title</Trans>;
+
+  //FILTER PART
+  const [servicePath, setServicePath] = React.useState(null);
+  const [type, setType] = React.useState(null);
+  const [date, setDate] = React.useState(null);
+  const filterMapper = {
+    servicePath: {
+      value: servicePath,
+      set: setServicePath
+    },
+    type: {
+      value: type,
+      set: setType
+    },
+    date: {
+      value: date,
+      set: setDate
+    }
+  };
+
+  // services
+  const [services, setServices] = React.useState([]);
+  const getServices = () => {
+    axios
+      .get(
+        (typeof env !== 'undefined' ? env.ANUBIS_API_URL : '') + 'v1/tenants/' + GeTenantData('id') + '/service_paths',
+        {}
+      )
+      .then((response) => {
+        setServices(response.data);
+      });
+  };
+
+  const GeTenantData = (type) => {
+    const tenantArray = tenantValues.filter((e) => e.id === thisTenant);
+    if (type === 'name') {
+      return tenantArray[0].name;
+    } else {
+      return tenantArray[0].id;
+    }
+  };
+
+  const getEntities = () => {
+    const queryParameters =
+      (type !== null ? '&type=' + type.type : '') +
+      (date !== null ? '&q=dateModified>=' + dayjs(date).toISOString() : '');
+
+    const headers =
+      servicePath !== null
+        ? {
+            'fiware-Service': GeTenantData('name'),
+            'fiware-ServicePath':
+              servicePath.path[servicePath.path.length - 1] === '/' ? servicePath.path : servicePath.path
+          }
+        : { 'fiware-Service': GeTenantData('name') };
+    axios
+      .get(env.ORION.replace('&orderBy=id', ',') + 'dateCreated,dateModified,*' + queryParameters, {
```

Comment:
mmm let's discuss about this

Reply:
> mmm let's discuss about this

i think that is ok for the retriving to have the url without parameters and then add the filtering on the policy form too

Reply:
1. as in the case of the policies, the url may be different (and so ideally it should be picked from the resource type `entity` for the tenant
2. the url may or may not contain arbitrary query. so you should strip the all query if any.

Reply:
@Valecant i refactored the resource type creation.
see:
- https://github.com/orchestracities/auth-management-ui/blob/entitiesTable%26Filters/.env.development
- https://github.com/orchestracities/auth-management-ui/blob/entitiesTable%26Filters/configuration-api/main/mongo/populateDB.js

so:
1. if entity resource type is available, load the entity endpoint from there and strip any query configured there to add the query required here.
2. if not available, load it from env.Orion and append + '/v2/entities?'